pkgs/collection/lib/src/unmodifiable_wrappers.dart

```diff
@@ -193,4 +193,12 @@ abstract mixin class UnmodifiableMapMixin<K, V> implements Map<K, V> {
   /// operations that change the map are disallowed.
   @override
   void clear() => _throw();
+
+  /// Throws an [UnsupportedError].
+  @Deprecated('This will be removed at a later date.')
+  set first(dynamic _) => _throw();
```

Comment:
Could we make the parameter type be `Object?` or `void`?

It should make no difference in a contravariant position, and I always get wary when I see `dynamic`.
(I think it makes not difference. With enum shorthands, I guess `Object?` will allow `.hash(...)`, so maybe just use `void`.)

Reply:
I switched to `Object?` to avoid the use of dynamic.